0.5/Dockerfile

```diff
@@ -59,8 +59,11 @@ ENV JANUS_VERSION=${JANUS_VERSION} \
 
 RUN groupadd -r janusgraph --gid=999 && \
     useradd -r -g janusgraph --uid=999 janusgraph && \
+    printf "\\ndeb http://ppa.launchpad.net/rmescandon/yq/ubuntu xenial main\\n\
```

Comment:
Wouldn' t be better to use pip for installation of yq?

Reply:
What would be the benefit of using pip over apt? I don't see pip installed in the current image so it would add another dependency.